config/queries.js

```diff
@@ -0,0 +1,195 @@
+import promise from 'bluebird';
+import moment from 'moment';
+import helper from '../helpers/helpers';
+import validator from '../middlewares/middlewares';
+
+const options = {
+	promiseLib: promise
+};
+const pgp = require('pg-promise')(options);
+const conString = 'postgres://postgres:rastafari@localhost:5432/questioner';
```

Comment:
Database credentials are very sensitive pieces of information. I'd be much better to have them stored in environment variables.

It'd also be better if you had a separate file responsible for connecting to your database since you will be connecting to it in multiple files. It will reduce redundancy.